django/template/library.py

```diff
@@ -83,8 +83,7 @@ def dec(func):
                     setattr(filter_func, attr, value)
                     # set the flag on the innermost decorated function
                     # for decorators that need it, e.g. stringfilter
-                    if hasattr(filter_func, "_decorated_function"):
```

Comment:
Without this we can end up setting the flag twice for unwrapped functions. Do we want to keep the if statement but switch `"_decorated_function"` for `"__wrapped__"`?

Reply:
What's the harm in setting the flag twice? Is that a performance issue?

Reply:
No harm. Not really. Just an observation that this is a change in behaviour.